README.md

```diff
@@ -1,66 +1,214 @@
+
+The HPE OneView Terraform SDK enables developers to easily build integrations and scalable solutions with HPE OneView and HPE Image Streamer.
+
+You can find the latest supported HPE OneView Terraform Provider SDK [here](https://github.com/HewlettPackard/terraform-provider-oneview/releases/latest)
+
+## What's New
+
+HPE OneView Terraform SDK library extends support of the SDK to OneView REST API version 2200 (OneView v5.50)
+
+Please refer to [notes](https://github.com/HewlettPackard/terraform-provider-oneview/blob/master/CHANGELOG.md) for more information on the changes , features supported and issues fixed in this version
+
+## Getting Started 
+
+## Installation and Configuration
 
 ## Installing `terraform-provider-oneview` with Go
 
-* Install Go 1.11. For previous versions, you may have to set your `$GOPATH` manually, if you haven't done it yet.
-* Install Terraform 0.11.x [from here](https://www.terraform.io/downloads.html) and save it into `/usr/local/bin/terraform` folder (create it if it doesn't exists). This provider DOES NOT SUPPORT Terraform 0.12 or above.
-* Download the code by issuing a `go get` command.
+HPE OneView SDK for Terraform can be installed from Source or Docker container installation methods. You can either use a docker container which will have the HPE OneView SDK for terraform installed or perform local installation manually.
+
+### Docker Setup
+The light weight containerized version of the HPE OneView SDK for Terraform is available in the [Docker Store](https://store.docker.com/community/images/hewlettpackardenterprise/hewlettpackardenterprise/hpe-oneview-sdk-for-terraform). The Docker Store image tag consist of two sections: <sdk_version-OV_version>
 
 ```bash
-# Download the source code for terraform-provider-oneview
-# and build the needed binary, by saving it inside $GOPATH/bin
-$ go get -u github.com/HewlettPackard/terraform-provider-oneview
+# Download and store a local copy of oneview-sdk-for-terraform and use it as a Docker Image.
+$ docker pull hewlettpackardenterprise/hpe-oneview-sdk-for-terraform:v1.6.0-OV5.5
+# Run docker commands below given, which  will in turn create a sh session 
+# where you can create files, issue commands and execute the examples.
+$ docker run -it docker pull hewlettpackardenterprise/hpe-oneview-sdk-for-terraform:v1.6.0-OV5.5 /bin/sh
+```
 
-# Copy the binary to have it along the terraform binary
-$ mv $GOPATH/bin/terraform-provider-oneview /usr/local/bin/terraform
+### Local Setup
+
+Local installation requires
+- Installing Go
+```bash 
+$ apt-get install build-essential git wget
+$ wget https://dl.google.com/go/go1.11.3.linux-amd64.tar.gz
+
+#unzip and untar the file 
+$ tar -zxvf "go1.11.linux-amd64.tar.gz", 
+
+# move it to /usr/local/ and create directory for Go.
+$ mv go1.11.3.linux-amd64.tar.gz /usr/local/ 
```

Comment:
You shoud be moving the untar file not the archive file here

Reply:
done